modules/features/endofyear/src/main/java/au/com/shiftyjelly/pocketcasts/endofyear/StoriesViewModel.kt

```diff
@@ -108,7 +114,7 @@ class StoriesViewModel @Inject constructor(
                     xStartOffsets = List(numOfStories) { getXStartOffsetAtIndex(it) },
                     widths = storyLengthsInMs.map { it / totalLengthInMs.toFloat() },
                 ),
-                userTier = settings.userTier,
+                userTier = currentUserTier,
```

Comment:
Instead of storing `currentUserTier` as a field, could we pass it as an argument in `updateState`? It looks like we only call `updateState` from one place (where we also set the `currentUserTier` field, so I think this would be an easy way to avoid having a field variable. Feel free to leave as-is if you prefer though.

Reply:
Nice pick! Suggestion applied in d0ffd6c9ee1a10863613ab86f8862b9bd2947fbe